config/default.js

```diff
@@ -0,0 +1,189 @@
+				org: process.env.NXB_GITHUB_ORG,
+				repo: process.env.NXB_GITHUB_REPO,
+				auth: process.env.NXB_GITHUB_TOKEN
+			},
+			cmd_prefix: "!",
+			commands_gitter: {
+				list: {
+					not_elevated: true,
+					run: function(user, args, gitter, irc) {
+						var users = irc.getUsers();
+						var msg = "";
+						if (users.length <= 1) {
+							msg = "[GitterBridge] There are no users online.";
+						} else {
+							users.map(function(value, index, array) {
+								if (value === irc.getUsername()) {
+									users.splice(index, 1);
+								}
+							});
+							msg = "[GitterBridge] There are " + users.length + " User(s) online: " + users.join(", ");
+						}
+						gitter.say(msg);
+					}
+				},
+				whois: {
+					not_elevated: true,
+					run: function(user, args, gitter, irc) {
+						irc.whois(args[0], function(username, whoisinfo) {
+							gitter.say("[GitterBridge] Whois for " + username + ":\n```\n" + JSON.stringify(whoisinfo, null, "\t") + "\n```");
+						}.bind(this, user));
+					}
+				},
+				xcloud: {
+					not_elevated: true,
+					run: function(user, args, gitter, irc) {
+						var msg = "XCloud Migration Guide: http://portal.new-xkit.com/assets/migration/xcloud-migration.html";
+						if (!args[0]) {
+							irc.send(msg);
+						} else {
+							irc.notice(args[0], msg);
+						}
+						gitter.say("[GitterBridge] Migration guide linked.");
+					}
+				},
+				chrome: {
+					not_elevated: true,
+					run: function(user, args, gitter, irc) {
+						var msg = "New XKit Download for Google Chrome: http://new-xkit-extension.tumblr.com/chrome";
+						if (!args[0]) {
+							irc.send(msg);
+						} else {
+							irc.notice(args[0], msg);
+						}
+						gitter.say("[GitterBridge] Chrome download linked.");
+					}
+				},
+				firefox: {
+					not_elevated: true,
+					run: function(user, args, gitter, irc) {
+						var msg = "New XKit Download for Mozilla Firefox: http://new-xkit-extension.tumblr.com/firefox";
+						if (!args[0]) {
+							irc.send(msg);
+						} else {
+							irc.notice(args[0], msg);
+						}
+						gitter.say("[GitterBridge] Firefox download linked.");
+					}
+				},
+				edump: {
+					not_elevated: true,
+					run: function(user, args, gitter, irc) {
+						var msg = "How to create an extension dump:\n" +
+						"1.) Go to your XKit Control Panel\n" +
+						"2.) Go to the 'Other' tab\n" +
+						"3.) Click on 'Export Configuration'\n" +
+						"4.) Depending on what we ask for, click either 'Extension Info Export' (this is what we need most of the time) or 'Full Configuration Export'\n" +
+						"5.) Copy the https://gist.github.com/ link it generates and give it to us\n";
+						if (!args[0]) {
+							irc.send(msg);
+						} else {
+							irc.notice(args[0], msg);
+						}
+						gitter.say("[GitterBridge] Extension dump guide sent.");
+					}
+				},
+				help: {
+					not_elevated: true,
+					run: function(user, args, gitter, irc) {
+						gitter.say(
+							"#Commands\n" +
```

Comment:
You know what? I absolutely didn't think of that, good idea.
Probably gonna do that in another PR after this one though.